GulpStart/index.html

```diff
@@ -9,8 +9,8 @@
 
     <link
```

Comment:
As far as I know, you attach foundation styles via gulp? Am I right?

Reply:
fixed: I removed the link